flusher.go

```diff
@@ -439,6 +456,68 @@ func resolveEndpoint(endpoint string) (string, error) {
 	return origURL.String(), nil
 }
 
+func (s *Server) flushTraces() {
+	traces := s.TraceWorker.Flush()
+
+	var finalTraces []*DatadogTraceSpan
+	traces.Do(func(t interface{}) {
+		if t != nil {
+			span, ok := t.(ssf.SSFSample)
+			if ok {
```

Comment:
nitpick: to reduce indentation, we can instead `if !ok { return }` and then unindent the successful case

also if you'd rather not do that, we can still combine the assertion and condition together: `if span, ok := t.(ssf.SSFSample) ; ok {`